recipes-support/mstpd/mstpd_git.bb

```diff
@@ -4,15 +4,11 @@ LICENSE = "GPLv2"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=4325afd396febcb659c36b49533135d4"
 
 SRC_URI = "\
-  git://github.com/mstpd/mstpd.git;protocol=https \
-  file://0001-libnetlink-introduce-rta_nest-and-u8-u16-u64-helpers.patch \
-  file://0002-libnetlink-Set-NLA_F_NESTED-in-rta_nest.patch \
-  file://0003-libnetlink-Add-helper-to-add-a-group-via-setsockopt.patch \
-  file://0004-WIP-set-CIST-MSTI-port-states-to-the-newly-added-per.patch \
+  git://github.com/bisdn/mstpd.git;branch=0.1.0+bisdn1;protocol=https \
   file://bridge-stp.conf"
 
-PV = "0.0.8+git${SRCPV}"
-SRCREV = "4d98707b9047a74bc473345875e60ca5a24f0c96"
+PV = "0.1.0+bisdn1"
```

Comment:
mstpd defines the package version in https://github.com/mstpd/mstpd/blob/833b2f2803e429206bee93e4ddd7c64e5f25f5e3/configure.ac#L80 as:

```
PACKAGE_VERSION="${PACKAGE_VERSION}-${PACKAGE_BUILD}"
```

Since PACKAGE_BUILD is not defined I think we will always end up with the version having a trailing dash '-'